### DISCUSSION TITLE
Is it possible to handle double clicked event in a console program?

### DISCUSSION THREAD

**Discussion by JasonWei512** [HavenDV/H.NotifyIcon#51] [Q&A] [answered]
First of all, thank you for creating this great library!
I am using the `H.NotifyIcon.Core` library in a console program. I want to execute some code when user double clicks the tray icon. Is it possible in the current version?

**Comment by HavenDV** [comment, marked as answer]
At the moment there is no easy way, you can do this:
```
TrayIcon.MessageWindow.MouseEventReceived += OnMouseEvent;
```
OnMouseEvent:
https://github.com/HavenDV/H.NotifyIcon/blob/master/src/libs/H.NotifyIcon.Shared/TaskbarIcon.MouseEvents.cs#L68-L116

I will also accept PR if you add the events you want to the TrayIcon class directly
